Fabric.Terminology.API/Modules/ValueSetMetadataModule.cs

```diff
@@ -197,6 +197,24 @@ public class ValueSetMetadataModule : SwaggerMetadataModule
                     TagsFactory.GetValueSetTag()
                 });
 
+            this.RouteDescriber.DescribeRouteWithParams(
+                "CompareValueSets",
+                "Compares two or more ValueSets",
+                "Compares two or more ValueSets",
+                new[]
+                {
+                    new HttpResponseMetadata<ValueSetApiModel> { Code = 200, Message = "OK" },
```

Comment:
These initializers are repetitive. Can you use a common property or method for these very common response types?

Reply:
Agree.  I'll look at simplifying these when we introduce the swagger UI project.